internal/test/test.go

```diff
@@ -0,0 +1,102 @@
+package test
+
+import (
+	"encoding/json"
+	"net/url"
+
+	"github.com/freerware/negotiator/internal/header"
+	"github.com/freerware/negotiator/representation"
+)
+
+func BuildVariant() VariantBuilder {
```

Comment:
add comments for all types and methods in this file. change to `RepresentationBuilder`.

Reply:
done!